app/src/features/auth/components/auth/forms/sign-in-form.tsx

```
@@ -0,0 +1,337 @@
+        <form.AppForm>
+            <form
+                className={cn("grid w-full gap-6", className, classNames?.base)}
+                noValidate={isHydrated}
+                onSubmit={(event) => {
+                    event.preventDefault();
+                    event.stopPropagation();
+                    form.handleSubmit();
+                }}
+            >
+                <form.AppField
+                    children={(field) => (
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)